pkg/services/authn/authnimpl/usage_stats.go

```diff
@@ -0,0 +1,54 @@
+package authnimpl
+
+import (
+	"context"
+
+	"github.com/grafana/grafana/pkg/services/authn"
+	"github.com/grafana/grafana/pkg/setting"
+)
+
+func (s *Service) getUsageStats(ctx context.Context) (map[string]interface{}, error) {
+	m := map[string]interface{}{}
+
+	// Add stats about auth configuration
+	authTypes := map[string]bool{}
+	authTypes["basic_auth"] = s.cfg.BasicAuthEnabled
+	authTypes["ldap"] = s.cfg.LDAPEnabled
+	authTypes["auth_proxy"] = s.cfg.AuthProxyEnabled
+
+	for authType, enabled := range authTypes {
+		enabledValue := 0
+		if enabled {
+			enabledValue = 1
+		}
+		m["stats.auth_enabled."+authType+".count"] = enabledValue
```

Comment:
this is the same stats as above in the socialservice, so this is for the new service, correct?
otherwise we would overwrite the stats

Reply:
these will not overwrite the stats as these only cover LDAP, Auth Proxy and Basic Auth. 
The social service is handling only oauth enabled so the keys don't overlap